packages/api/src/index.ts

```diff
@@ -16,13 +16,29 @@ import {
 } from './types'
 import { Web3Middleware } from './web3Middleware/index'
 import { normalizeConfig, normalizeNetworkConfig } from './utils/index'
-import dataFeedsRouterConfig from './dataFeedsRouter.json'
+import axios from 'axios'
+
+async function getDataFeedsRouterConfig (): Promise<
+  RouterDataFeedsConfig | null
+> {
+  return await axios
+    .get(
+      'https://raw.github.com/witnet/data-feeds-explorer/main/packages/api/src/dataFeedsRouter.json'
+    )
+    .then(res => {
+      return res.data
+    })
+    .catch(err => {
+      console.log('There was an error fetching the config file', err)
+      return null
+    })
+}
 
 async function main () {
   const mongoManager = new MongoManager()
   const db = await mongoManager.start()
-  const dataFeeds = readDataFeeds()
-  const networksConfig = readNetworks()
+  const dataFeeds = await readDataFeeds()
+  const networksConfig = await readNetworks()
```

Comment:
Can we use [`Promise.allSettled`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) to reduce waiting time?